package.json

```diff
@@ -150,6 +151,7 @@
     "is-valid-domain": "0.1.2",
     "lint-staged": "10.4.0",
     "lodash": "4.17.21",
+    "lodash.debounce": "4.0.8",
```

Comment:
I checked this package and seems to be published by the same people as lodash is, and it gets 16mln weekly downloads (vs whole lodash 43mln), so in this regard feels safe.

But it was last updated 6y ago, so I am not sure if this is maintained and kept up to date with lodash changes (especially security vulnerabilities).

We should check.

Also I am curious about bundling and tree shaking. Is there a chance that we can import just debounce from the lodash package and not have the whole lodash in our bundle? It would be nice (but then we would need an eslint rule for forbidding lodash apart from debounce or some other measure to make sure it doesn't come back 😄).

Reply:
Yeah, this package is from the same author, so it is safe. 
I checked it, and it has minor differences in helpers like `toObject` and `toNumber`. So it's a stable version and I guess it ok to use it

Reply:
I also checked - this is pretty much the same package. But would be cool to also verify it doesn't have any of the security problems lodash had, @perunt

Reply:
One problem regarding a potential supply chain attack here is related to how fast lodash maintainers will update this package in case lodash gets a new vulnerability discovered in this function.

I assume they will update the main package very quickly, but do they even remember about this one if it wasn't touched in 6y?

Reply:
searched for the latest publicly known vulnerabilities of lodash and didn't find anything related to `lodash.deboubce` or `toNumber`, `toObject`. 
From my perspective it looks safe